dataprofiler/labelers/data_processing.py

```diff
@@ -2055,14 +2155,19 @@ def _save_processor(self, dirpath):
         ) as fp:
             json.dump(params, fp)
 
-    def process(self, data, labels=None, label_mapping=None, batch_size=None):
+    def process(
+        self,
+        data: np.ndarray,
+        results: Dict,
+        label_mapping: Dict[str, int],
```

Comment:
Renamed `labels` to `results` to be consistent with other postprocessors `process` and the superclass

Reply:
good call